### ISSUE TITLE
switch to counts rather than binary inclusions for words in message

### ISSUE THREAD

**Issue by ClimbsRocks** [ClimbsRocks/nlpSentiment#12]
as part of this, make sure to force exclamation points to be included

**Comment by ClimbsRocks** [comment]
this helps us tremendously to learn the training set for ATC, but apparently introduces a lot of overfitting to those training sets, as it decreases our accuracy against the test data.
